modules/pel/peltool/parse_user_data.py

```diff
@@ -40,25 +40,35 @@ def parse(self) -> str:
         else:
             value = self.parseCustom()
 
+        # Catch if value is None, otherwise python crashes
+        if value == None:
+            value = ("PROBLEM with UNRECOGNIZED value from parser creatorID={} compID={} subType={} version={}"
```

Comment:
could just say 'Parser returned a value of None' so it says what the real problem is.

Reply:
Sounds good ...